legion/engine/llri/llri.hpp

```diff
@@ -52,6 +53,9 @@
 #define LLRI_NAMESPACE llri
 #endif
 
+#define LLRI_TIMEOUT_NONE 0u
+#define LLRI_TIMEOUT_INFINITE 0xFFFFFFFFu
```

Comment:
does this actually end up disabling timeout, or is this just the maximum timeout? cuz if so, LLRI_TIMEOUT_MAX might be a better name

Reply:
Depends on the implementation. In DirectX it does, Vulkan does not give this same guarantee, but it does guarantee that the time the function takes is finite.

Reply:
yeah then i'd say call it LLRI_TIMEOUT_MAX, which may be infinite, or might not be